code-challenges.test.js

```diff
@@ -15,35 +20,180 @@
 
 // a) Create a test with an expect statement using the variable provided. HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
-const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+
 // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
 
-// b) Create the function that makes the test pass.
+
+describe('arrayShuffler', () => {
+    const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+    const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+    it("takes in an array, removes the first item from the array and shuffles the remaining content", () => {
+        expect(arrayShuffler(colors1)).toEqual(expect.arrayContaining(["yellow", "blue", "pink", "green"]))
+        expect(arrayShuffler(colors2)).toEqual(expect.arrayContaining(["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]))
+    })
+  })
 
 
+// b) Create the function that makes the test pass.
+// Create a function that takes in an array, removes the first item from the array and shuffles the remaining content.
+
+// ***PSEUDOCODE***
+//create a function named arrayShuffler
+//parameter is array
+//remove the first element and shuffle the rest
+//originally tried to use .slice method but couldnt figure out the syntax
+// using .shift method removes first element in the array
+// use math.floor/math.random method for possible solution as that will shuffle 
+//the new array output
```

Comment:
We are almost there, but there are still a few pieces missing in the pseudocode.  Based on your pseudocode alone, if I were to code this out here is what I came up with: 
`const arrayShuffler = (array) => {
  array.shift()
  let randomNum = Math.floor(Math.random())
  let newArray = array[randomNum]
  return newArray
}`
What is key thing is missing in the above function that you have written in your function but not your pseudocode?  Hint: take another look at line 56.  Your updated function looks great, just want to see your pseudocode match it.